.gitignore

```diff
@@ -1,3 +1,5 @@
+jupyterlab/labbox_ephys_widgets_jp/src/extensions
```

Comment:
No we gitignore the jupyterlab/labbox_ephys_widgets_jp/src/extensions

and so we don't duplicate code in the repo!